<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelLiveCam">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrimoFrame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>861</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="bottoneAvviaWebcam">
      <property name="text">
       <string>Avvia Webcam</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bottoneStopWebcam">
      <property name="text">
       <string>Stop Webcam</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bottoneCatturaMovimento">
      <property name="text">
       <string>Cattura Movimento</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bottonePrimoFrame">
      <property name="text">
       <string>Primo Frame</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>247</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineaInizioMovimento"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Inizio Movimento:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Fine Movimento:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineaFineMovimento"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineaDurataMovimento"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Durata Movimento:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="bottoneCaricaVideo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Carica Video</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
